src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --is-active-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.moyo_header {
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.home_link {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav_link {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  cursor: pointer;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding